packages/consent-manager/src/components/FallbackComponent.tsx

```diff
@@ -6,6 +6,7 @@ import { useIntegration } from '../integrations'
 export const FallbackComponent: React.FC<FallbackComponentProps> = ({
   integrationId,
   fallbackUrl,
+  ...props
```

Comment:
Does that change our TS typings or was the test id accepted beforehand already but just not passed on? Usually not a big fan of spreading "everything we got" to the child, but I guess it makes sense here